src/coreclr/src/jit/gentree.cpp

```diff
@@ -10846,8 +10849,8 @@ void Compiler::gtDispLeaf(GenTree* tree, IndentStack* indentStack)
                             fieldVarDsc->PrintVarReg();
                         }
 
-                        if (fieldVarDsc->lvTracked && fgLocalVarLivenessDone && // Includes local variable liveness
-                            ((tree->gtFlags & GTF_VAR_DEATH) != 0))
+                        if (fieldVarDsc->lvTracked && fgLocalVarLivenessDone &&
+                            tree->AsLclVar()->IsLastUse(i - varDsc->lvFieldLclStart))
```

Comment:
It should probably be `AsLclVarCommon` or it will fail for `GT_LCL_FLD`.

Reply:
I added a check for `tree->IsMultiRegLclVar()`, as that's a more precise condition that it has the last-use bits.